MiguelAValdesG/HackerRank/springBootAPIRest/src/main/java/com/bootcamp/springBootAPIRest/service/impl/AnagramsServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.bootcamp.springBootAPIRest.service.impl;
+
+import com.bootcamp.springBootAPIRest.service.AnagramsService;
+import com.bootcamp.springBootAPIRest.utils.Constants;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+
+@Service
+public class AnagramsServiceImpl implements AnagramsService {
+
+  static boolean isAnagram(String firstWord, String secondWord) {
+    if (firstWord.isEmpty() || secondWord.isEmpty() || firstWord.length() != secondWord.length()) {
+      return false;
+    } else {
+      char[] firstWordArray = firstWord.toLowerCase().toCharArray();
+      char[] secondWordArray = secondWord.toLowerCase().toCharArray();
+
+      Arrays.sort(firstWordArray);
+      Arrays.sort(secondWordArray);
+      return Arrays.equals(firstWordArray, secondWordArray);
+    }
+  }
+
+  public String anagrams(String firstWord, String secondWord) {
+    boolean response = isAnagram(firstWord, secondWord);
```

Comment:
Fix Indentation

Reply:
I didn't see any problem with the indentation in this part of code, in fact, I "ran" the automatic reformat in InteliJ (ctrl + alt + l), but everything was ok and didn't have any change.